python/ray/serve/_private/long_poll.py

```diff
@@ -214,6 +223,22 @@ def _get_num_notifier_events(self, key: Optional[KeyType] = None):
         else:
             return sum(len(events) for events in self.notifier_events.values())
 
+    def count_bytes_returned(listen_for_change_func):
```

Comment:
add type hints for expected return value of `listen_for_change_func`

nit: it's also a bit of an abstraction leak to name it "listen_for_change_func"

Reply:
Thanks for raising this, but due to https://github.com/ray-project/ray/pull/39777#issuecomment-1731899162, I'll have to change the overall implementation.